src/js/components/HashMapDisplay.js

```diff
@@ -31,7 +33,14 @@ class HashMapDisplay extends React.Component {
   }
 
   getItems() {
-    const { hash, headingLevel, renderKeys } = this.props;
+    const {
+      hash,
+      headingLevel,
+      renderKeys,
+      onEditClick,
+      keyPath,
+      emptyValue
+    } = this.props;
 
     return Object.keys(hash).map((key, index) => {
       let value = hash[key];
```

Comment:
Maybe this could be improved with 
`let value = hash[key] || "";`
because if it is undefined or null it may throw an error with the join

Reply:
I don't this is needed because we are mapping over its keys. So when when access it's key it can't be undefined.